test/common/scale_up_replicas_rhsso.go

```diff
@@ -39,6 +40,10 @@ func TestReplicasInUserSSO(t TestingTB, ctx *TestingContext) {
 		t.Fatalf("failed to get RHMI instance %v", err)
 	}
 
+	if integreatlyv1alpha1.IsRHOAMMultitenant(integreatlyv1alpha1.InstallationType(inst.Spec.Type)) {
```

Comment:
Do you think it would be cleaner to add another TestCase to `SCALABILITY_TESTS` instead, something like common and managed-api only?